code/modules/mob/living/carbon/human/species/vox.dm

```diff
@@ -160,13 +160,13 @@
 	icon_template = 'icons/mob/human_races/r_armalis.dmi'
 
 	has_organ = list(
-		"heart" =    /obj/item/organ/internal/heart,
-		"lungs" =    /obj/item/organ/internal/lungs/vox,
-		"liver" =    /obj/item/organ/internal/liver,
-		"kidneys" =  /obj/item/organ/internal/kidneys,
-		"cortical stack" =    /obj/item/organ/internal/brain/vox,
-		"eyes" =     /obj/item/organ/internal/eyes, //Default darksight of 2.
-		)												//for determining the success of the heist game-mode's 'leave nobody behind' objective, while this is just an organ.
+		ORGAN_HEART	   = /obj/item/organ/internal/heart/vox,
+		ORGAN_LUNGS	   = /obj/item/organ/internal/lungs/vox,
+		ORGAN_LIVER	   = /obj/item/organ/internal/liver/vox,
+		ORGAN_KIDNEYS  = /obj/item/organ/internal/kidneys/vox,
+		ORGAN_BRAIN	   = /obj/item/organ/internal/brain/vox,
+		ORGAN_EYES	   = /obj/item/organ/internal/eyes/vox, //Default darksight of 2.
```

Comment:
No mechanical changes here other than a 1.6x alcohol intensity from the liver, and all organs being sterile. This is just so that the sprites and descriptions are consistent between species.